### PR TITLE
added users role

### PR DESCRIPTION
**Pull request by tsehelnyk**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/controller/LogOutController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    private static final String COOKIE_NAME = "MATE";
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        HttpSession session = req.getSession();
+        session.setAttribute("user_id", null);
+
+//        resp.addCookie(new Cookie(COOKIE_NAME, ""));
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/controller/LogOutController.java]: remove comment please

File: src/main/java/mate/academy/internetshop/controller/LogOutController.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+public class LogOutController extends HttpServlet {
+
+    private static final String COOKIE_NAME = "MATE";
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        HttpSession session = req.getSession();
+        session.setAttribute("user_id", null);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/controller/LogOutController.java]: req.getSession.invalidate();

File: src/main/webapp/WEB-INF/views/denied.jsp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
```
Comment (Tarabarnik) [inline src/main/webapp/WEB-INF/views/denied.jsp]: remove comment please

File: src/main/java/mate/academy/internetshop/controller/LogOutController.java (hunk 1 above)
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/controller/LogOutController.java]: done

File: src/main/java/mate/academy/internetshop/controller/LogOutController.java (hunk 2 above)
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/controller/LogOutController.java]: done

File: src/main/webapp/WEB-INF/views/denied.jsp (hunk 3 above)
Comment (tsehelnyk) [reply src/main/webapp/WEB-INF/views/denied.jsp]: comments removed

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**